.github/workflows/scripts/baking/bake-props.lua

```diff
@@ -0,0 +1,209 @@
+--------------------------------------------------------------------------------
+-- Supreme Commander mod automatic unit wiki generation script for Github wikis
+-- Copyright 2021-2022 Sean 'Balthazar' Wheeldon                      Lua 5.4.2
+--------------------------------------------------------------------------------
+
+--[{ ---------------------------------------------------------------------- ]]--
+--[[ Inputs -- NOTE: Mod input files must be valid lua                      ]]--
+--[[ ---------------------------------------------------------------------- ]]--
+local OutputDirectory = "D:/faf-development/fa.wiki/"
+local WikiGeneratorDirectory = "D:/faf-development/BrewWikiGen/"
+local FADirectory = "D:/faf-development/fa/"
+
+-- This section deals with overriding the OutputDirectory and WikiGeneratorDirectory if required
```

Comment:
At a glance the comment looks like it describes the function (that's how it is for all other lua in this repo). I think we should use a region and add a newline.
```suggestion
--#region This section deals with overriding the OutputDirectory and WikiGeneratorDirectory if required by command-line arguments

```